newIDE/app/src/stories/index.js

```diff
@@ -2457,6 +2459,142 @@ storiesOf('ParameterFields', module)
         />
       )}
     />
+  ))
+  .add('RelationalOperatorField (string)', () => (
```

Comment:
maybe this was the opportunity to extract `ParameterFields` to its own story file? 😇

Reply:
Pain me to tell that but yeah that's a good opportunity!